src/components/Models/Model/Model.component.tsx

```diff
@@ -67,10 +67,8 @@ export const Model: FC<ModelProps> = ({
 
     if ((node as Mesh).isMesh) {
       node.castShadow = true;
-    }
-
-    if (node.type === 'SkinnedMesh') {
       node.receiveShadow = true;
+      node.frustumCulled = false;
```

Comment:
As attachments are Mesh, lets not differentiate between mesh and skinnedmesh.

Disabling frustum culling due to a bug I noticed where eyes do get culled, is a classic three problem I have encountered before. Will try and get a screenshot of it though!